backend/src/forecastbox/api/routers/job.py

```diff
@@ -425,7 +427,7 @@ def _build_zip() -> tuple[bytes, str]:
         )
 
 
-@router.get("/{job_id}/results/{dataset_id}")
+@router.get("/{job_id}/results")
```

Comment:
cc @liefra -- this PR marks a route deprecated. If it gets merged, please at your convenience switch from dataset_id being a path-param to a query-param. I will then eventually delete the deprecated routes